ext/ffi/ffi.c

```diff
@@ -2946,9 +2958,16 @@ ZEND_METHOD(FFI, cdef) /* {{{ */
 	if (lib) {
 		handle = DL_LOAD(ZSTR_VAL(lib));
 		if (!handle) {
-			zend_throw_error(zend_ffi_exception_ce, "Failed loading '%s'", ZSTR_VAL(lib));
+			err = GET_DL_ERROR();
+#ifdef PHP_WIN32
+			php_win32_error_msg_free(err);
+#else
+			zend_throw_error(zend_ffi_exception_ce, "Failed loading '%s' (%s)", ZSTR_VAL(lib), err);
```

Comment:
This looks like a use-after-free on Windows.

Reply:
I do not think that there was a use-after-free, but I was not throwing an error on windows. I have updated my PR, please have a look.